### PR TITLE
committing a tutorial outline

### PR DESCRIPTION
**Pull request by meganmason**

### CODE REVIEW

**Comment by meganmason** [comment]
Wondering if the QualityControl test failed because I either didn't clear my outputs or didn't run the notebooks from top to bottom before committing.

**Review by scottyhq — CHANGES_REQUESTED** [review]
> Wondering if the QualityControl test failed because I either didn't clear my outputs 

Yep, you'll have to 'restart kernal and clear all outputs', save and commit the empty notebook


Also don't forget to add the new files to the table of contents file `_toc.yaml` (https://github.com/snowex-hackweek/website2022/blob/main/CONTRIBUTING.md#contributing-tutorials)

**Comment by meganmason** [comment]
Awesome, will do. Also, I'm okay to leave this PR open for a bit as I continue to make changes today. Thanks @scottyhq

**Comment by meganmason** [comment]
hey @scottyhq and @jomey, I created a new branch and PR for my toc.yml updates. My goal was to be more up-to-date with the Main, and push a small change. Do you think if that PR gets accepted this one will pass the required tests?

**Comment by scottyhq** [comment]
You'll actually want to add those changes to the _toc.yml here in your `core-data` branch. that way jupyterbook knows where to put the notebooks you're adding here in relation to other pages on the website.

**Comment by meganmason** [comment]
Okay, will do! Wishful thinking on my end :) 

I'm guessing the other PR I issued will just be rejected and I can delete that branch?

File: book/_toc.yml [hunk 1]
```diff
@@ -46,6 +46,12 @@ parts:
             - file: tutorials/database/5_plot_raster_example
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
+       - file: tutorials/data-overview/index
+         title: SnowEx Mission and Dataset Descriptions
+         sections:
+           - file: tutorials/data-overview/01_campaigns-overview
+           - file: tutorials/data-overview/02_ground-based-specifics
+           - file: tutorials/data-overview/03_snowexsql_db_querying
```
Comment (scottyhq) [inline book/_toc.yml]:
```suggestion
        - file: tutorials/core-datasets/index
          title: SnowEx Mission and Dataset Descriptions
          sections:
            - file: tutorials/core-datasets/01_campaigns-overview
            - file: tutorials/core-datasets/02_ground-based-specifics
            - file: tutorials/core-datasets/03_snowexsql_db_querying 
```

Comment (scottyhq) [reply book/_toc.yml]: ☝️ you can just hit the 'commit suggestion' to fix indent level and path names and it should then try to build

Comment (meganmason) [reply book/_toc.yml]: I missed your note about hit 'commit suggestion'. But did go back in and make the changes on the branch, commited, and pushed.

Comment (meganmason) [reply book/_toc.yml]: I keep confusing the word 'comit' and 'coment' lately.... so I don't think registered to select the 'commit suggestion' box. 😂

Comment (scottyhq) [reply book/_toc.yml]: all good. the spacing is still off by one though, so you need to indent still. Also you'll need to add a file `tutorials/core-datasets/index.md` which can just have the title as content: 
```
# SnowEx Mission and Dataset Descriptions

```

**Comment by meganmason** [comment]
hmm, not doing too well on the checks (0/4). 😬 When I click the red 'Changes requested' link I don't see anything new that needs attention.  If possible, I'm interested in getting this merged before continuing to develop content.

**Review by meganmason — COMMENTED** [review]
Little confused by this part. The green (upper right) button says 'Review changes', but everything is green and I don't see something that looks like a 'requested change'?

**Comment by jomey** [comment]
Looks like your branch was not up to date with `main` when you created your branch.
Want to do a `git merge main` when you are on this branch and push anew?
Some failures look unrelated to your changes and due to having 'old' main content.

You can see how your branch is behind in this view:
https://github.com/snowex-hackweek/website2022/branches

image

**Comment by meganmason** [comment]
Thanks, @jomey. I'm up to date with the main now, but still, scoring a 0 on the checks? Notebook outputs were all cleared before doing a `git push`
Screen Shot 2022-06-29 at 12 39 34 PM

**Comment by scottyhq** [comment]
@meganmason you might have missed my comment earlier, copying here

all good. the spacing is still off by one though, so you need to indent still. Also you'll need to add a file `tutorials/core-datasets/index.md` which can just have the title as content:

```
# SnowEx Mission and Dataset Descriptions

```

File: book/_toc.yml [hunk 2]
```diff
@@ -46,6 +46,12 @@ parts:
             - file: tutorials/database/5_plot_raster_example
             - file: tutorials/database/6_exporting_data
             - file: tutorials/database/7_wrap_up
+       - file: tutorials/core-datasets/index
+         title: SnowEx Mission and Dataset Descriptions
+         sections:
+           - file: tutorials/core-datasets/01_campaigns-overview
+           - file: tutorials/core-datasets/02_ground-based-specifics
+           - file: tutorials/core-datasets/03_snowexsql_db_querying
```
Comment (scottyhq) [inline book/_toc.yml]:
```suggestion
        - file: tutorials/core-datasets/index
          title: SnowEx Mission and Dataset Descriptions
          sections:
            - file: tutorials/core-datasets/01_campaigns-overview
            - file: tutorials/core-datasets/02_ground-based-specifics
            - file: tutorials/core-datasets/03_snowexsql_db_querying 
```

**Comment by meganmason** [comment]
after @jomey's comment, I probably should have done a `git pull` in my `core-data` branch? I think that's what git wants now since it won't let me `git push` my new index.md file. 
Screen Shot 2022-06-29 at 1 00 47 PM

**Comment by scottyhq** [comment]
Right, that message comes up b/c we just committed that last change (Update book/_tock.yml) on GitHub, but you didn't have that change locally. So `git pull` then `git push`

**Comment by meganmason** [comment]
cool, will do. Just always have to convince myself that a `git pull` on a branch won't overwrite all the branch changes. Since branch changes have been committed I should be assured they won't go away where yah?

**Comment by scottyhq** [comment]
Awesome! Almost there. Under the 'Details' for the checks you'll see the following:
`/home/runner/work/website2022/website2022/book/tutorials/core-datasets/01_campaigns-overview.ipynb:60002: WARNING: Non-consecutive header level increase; 1 to 3 [myst.header]` 

Jupyter Book is picky about headings for the table of contents. You can't have: 
```
# Title
### Subtitle
```

It needs to be: 
```
# Title
## Subtitle
```

**Comment by meganmason** [comment]
awh, I recall that from last year! Thought I was being careful.

**Comment by scottyhq** [comment]
Woohoo! You can now see how things look here https://deploy-preview-62--snowex2022.netlify.app/tutorials/core-datasets/index.html . The remaining failing check is for empty cells in the notebook `notebook cell 7: empty cell` I actually think we're being too strict there, so I'll make a note to change that. But if you remove those placeholder cells for now we can merge

**Comment by meganmason** [comment]
yay, some green! seems to think I still don't have clean notebooks... I'll be MIA for a bit, but afterward will jump back on!

**Comment by meganmason** [comment]
copy! I wondered if blank cells would be an issue. I'll address that, but not yet. Thanks, @scottyhq !

**Comment by meganmason** [comment]
Hey @scottyhq, I see I have some conflicts in the _toc.yml, but I'm not sure how to interpret what git is suggesting.
Screen Shot 2022-07-01 at 8 13 00 AM

**Comment by meganmason** [comment]
hey @jomey , I've been watching to make sure I keep up with main. I did a `git merge main, git pull, git push` again and am not convinced I'm up to date based on the cool branch tool you showed me. I read this as I'm still 7 commits behind. Maybe it never goes to zero while a PR is open, but the goal is to get a little more up-to-date with the main, verses before when I had a "really old main". 

Screen Shot 2022-07-01 at 8 22 37 AM

**Comment by meganmason** [comment]
we're in the green zone @scottyhq !! (Still adding content to notebooks, but it's the first time all checks have passed). I see 'changes requested' still, but don't see detail on them.

**Comment by scottyhq** [comment]
> 'changes requested' still, but don't see detail on them. 

That remains until I manually change to 'approved'.  

This looks great, thanks @meganmason ! Nice work loading the dataframe from a gist URL! It looks like the GIF isn't rendering in the preview. I know that the following syntax works (from an icesat-2 hackweektutorial): 

`cloud bouncing and smiling`

https://github.com/ICESAT-2HackWeek/website2022/blob/10229090a94edb3e5734dbc00149b0502ad5396a/book/tutorials/cloud-computing/cloud-computing-tutorial.ipynb?short_path=41df864#L12


But feel free to merge this one whenever, (Green 'Squash and Merge' button) you can always start fresh with additional changes

**Review by scottyhq — APPROVED** [review]

**Comment by meganmason** [comment]
Fantastic - yes not complete yet, but progress is happening! I'll go ahead and merge and then start fresh when I pick this back up. Thanks for the beta on the gif.

**Pull request merged**